privacyidea/static/components/user/views/user.add.html

```diff
@@ -1,5 +1,15 @@
+<style type="text/css">
+  .css-form input.ng-invalid.ng-touched {
+    background-color: #FA787E;
+  }
+
+  .css-form input.ng-valid.ng-touched {
+    background-color: #78FA89;
+  }
+</style>
```

Comment:
Probably not :)
It has a temporary/hacky inline style to quickly show that form checking was/is still working.